### ISSUE TITLE
doc building

### ISSUE THREAD

**Issue by cjacoby** [pescadores/pescador#39]
I am trying to build the docs so I can edit them (looking at the real thing), but I am getting this error:

``` bash
make html
sphinx-build -b html -d _build/doctrees   . _build/html
Running Sphinx v1.4.8

Recursion error:
maximum recursion depth exceeded while calling a Python object

This can happen with very large or deeply nested source files.  You can carefully increase the default Python recursion limit of 1000 in conf.py with e.g.:
    import sys; sys.setrecursionlimit(1500)
make: *** [html] Error 1
```

Do you have any experience with this? I'm a noob to sphinx.
All I have done is
- install sphinx and numpydoc
- run make html in the doc directory.

**Comment by bmcfee** [comment]
Confirmed -- will look into it when i have some time.

**Comment by bmcfee** [comment]
I fixed this one in the dev/0.2 branch.  It works fine if we only bulid on py3 and use unittest's MagicMock directly.

**Comment by bmcfee** [comment]
fixed by merging #38
